public/js/components/Scopes.js

```diff
@@ -39,7 +39,7 @@ function getSpecialVariables(pauseInfo, parentName) {
     vars.push({
       name: "<exception>",
       path: parentName + "/<exception>",
-      contents: { value: thrown.toJS() }
+      contents: { value: thrown }
```

Comment:
thrown should be an immutable object. It works for me, so I don't know what edge case you've found. It doesn't make any sense that it wouldn't be an immut object but `returned` and `this` would be... I'll check to make sure it is working for me, but I wonder what you hit.